www/front_src/src/Resources/translatedLabels.ts

```diff
@@ -213,3 +213,7 @@ export const labelCalculationType = 'Calculation type';
 export const labelSelectAtLeastOneColumn =
   'At least one column must be selected';
 export const labelMaxDuration1Year = 'The duration must be lesser than a year';
+export const labelSelectedSizeExport = 'Selected Size Export';
```

Comment:
This label is not used here. Is it for test purpose ?

Reply:
No it's on my own, Bruno disagree this change.